spec/requests/get_open_schools_spec.rb

```diff
@@ -1,13 +1,21 @@
 require 'spec_helper'
 require 'json'
+require 'csv'
 
 RSpec.describe 'GET /open_schools' do
+  let(:open_school_count) do
+    CSV
+      .read(Dir.glob('tmp/*test-set*').pop, headers: true)
```

Comment:
I wonder if just hardcoding some test data is better here. This clearly isn't the most robust thing ever written 🙈